ci-scripts/copy_to_archive

```diff
@@ -33,42 +32,53 @@ fi
+if [ -f ${LICENSE_DIR}/${IMAGE_BASENAME}-*20*/license.manifest ]; then
+    echo "Archiving license.manifest file to $ARCHIVE_DIR"
+    mv ${LICENSE_DIR}/${IMAGE_BASENAME}-*20*/license.manifest $ARCHIVE_DIR
 fi
 
-# If there is an Intel directory
-if [ -d ${IMAGE_DIR}/${INTEL_ARCH} ]; then
-    echo "Archiving Intel image to $ARCHIVE_DIR"
-    mv ${IMAGE_DIR}/${INTEL_ARCH}/${IMAGE_BASENAME}-*rootfs.wic $ARCHIVE_DIR
-    mv ${IMAGE_DIR}/${INTEL_ARCH}/${IMAGE_BASENAME}-*rootfs.wic.bmap $ARCHIVE_DIR
-    mv ${IMAGE_DIR}/${INTEL_ARCH}/${IMAGE_BASENAME}-*rootfs.wic.bz2 $ARCHIVE_DIR
-    mv ${IMAGE_DIR}/${INTEL_ARCH}/${IMAGE_BASENAME}-*20*.swu $ARCHIVE_DIR
+# Copy if image files exist
+if [ -f ${IMAGE_DIR}/**/${IMAGE_BASENAME}-*rootfs.wic ]; then
+    echo "Archiving ${IMAGE_BASENAME}-rootfs.wic file to $ARCHIVE_DIR"
+    mv ${IMAGE_DIR}/**/${IMAGE_BASENAME}-*rootfs.wic $ARCHIVE_DIR
 fi
 
-# If there is an ARP directory
+if [ -f ${IMAGE_DIR}/**/${IMAGE_BASENAME}-*rootfs.wic.bmap ]; then
+    echo "Archiving ${IMAGE_BASENAME}-rootfs.wic.bmap file to $ARCHIVE_DIR"
+    mv ${IMAGE_DIR}/**/${IMAGE_BASENAME}-*rootfs.wic.bmap $ARCHIVE_DIR
+fi
 
-if [ -d ${IMAGE_DIR}/${ARP_ARCH} ]; then
-    echo "Archiving ARP image to $ARCHIVE_DIR"
-    mv ${IMAGE_DIR}/${ARP_ARCH}/${IMAGE_BASENAME}-*rootfs.wic $ARCHIVE_DIR
-    mv ${IMAGE_DIR}/${ARP_ARCH}/${IMAGE_BASENAME}-*rootfs.wic.bmap $ARCHIVE_DIR
-    mv ${IMAGE_DIR}/${ARP_ARCH}/${IMAGE_BASENAME}-*rootfs.wic.bz2 $ARCHIVE_DIR
-    mv ${IMAGE_DIR}/${ARP_ARCH}/${IMAGE_BASENAME}-*20*.swu $ARCHIVE_DIR
+if [ -f ${IMAGE_DIR}/**/${IMAGE_BASENAME}-*rootfs.wic.bz2 ]; then
+    echo "Archiving ${IMAGE_BASENAME}-rootfs.wic.bz2 file to $ARCHIVE_DIR"
+    mv ${IMAGE_DIR}/**/${IMAGE_BASENAME}-*rootfs.wic.bz2 $ARCHIVE_DIR
 fi
 
-# If there is an RPI directory
-if [ -d ${IMAGE_DIR}/${RPI_ARCH} ]; then
-    echo "Archiving RPI image to $ARCHIVE_DIR"
-    mv ${IMAGE_DIR}/${RPI_ARCH}/${IMAGE_BASENAME}-*rootfs.wic $ARCHIVE_DIR
-    mv ${IMAGE_DIR}/${RPI_ARCH}/${IMAGE_BASENAME}-*rootfs.wic.bmap $ARCHIVE_DIR
-    mv ${IMAGE_DIR}/${RPI_ARCH}/${IMAGE_BASENAME}-*rootfs.wic.bz2 $ARCHIVE_DIR
-    mv ${IMAGE_DIR}/${RPI_ARCH}/${IMAGE_BASENAME}-*20*.swu $ARCHIVE_DIR
+if [ -f ${IMAGE_DIR}/**/${IMAGE_BASENAME}-*20*.swu ]; then
+    echo "Archiving ${IMAGE_BASENAME}.swu file to $ARCHIVE_DIR"
+    mv ${IMAGE_DIR}/**/${IMAGE_BASENAME}-*20*.swu $ARCHIVE_DIR
+fi
+
+if [ -f ${IMAGE_DIR}/**/${IMAGE_BASENAME}-*rootfs.manifest ]; then
+    echo "Archiving ${IMAGE_BASENAME}-rootfs.manifest file to $ARCHIVE_DIR"
+    mv ${IMAGE_DIR}/**/${IMAGE_BASENAME}-*rootfs.manifest $ARCHIVE_DIR
 fi
 
 # If there is a Jetson TX2 directory
-if [ -d ${IMAGE_DIR}/${JETSON_ARCH} ]; then
+if [ -f ${IMAGE_DIR}/**/${IMAGE_BASENAME}-*.tegraflash.zip ]; then
     echo "Archiving Jetson TX2 image to $ARCHIVE_DIR"
-    mv ${IMAGE_DIR}/${JETSON_ARCH}/${IMAGE_BASENAME}-*.tegraflash.zip $ARCHIVE_DIR
+    mv ${IMAGE_DIR}/**/${IMAGE_BASENAME}-*.tegraflash.zip $ARCHIVE_DIR
```

Comment:
need to change from `${IMAGE_BASENAME}-*.tegraflash.zip` to `${IMAGE_BASENAME}-*20*.tegraflash.zip` because at the moment, the (dereferenced) symlink is also copied.

See e.g. https://pelux.io/jenkins/job/pelux-manifests_NIGHTLY_jetson-tx2-minimal/